src/DependencyInjection/SymfonycastsSassExtension.php

```diff
@@ -80,11 +86,39 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->scalarNode('binary')
                     ->info('The Sass binary to use')
                     ->defaultNull()
+                ->end()
+                ->arrayNode('sass_options')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->booleanNode('charset')
+                            ->info('Whether to include the charset declaration in the generated Sass. (default: true)')
+                            ->defaultValue(true)
+                        ->end()
+                        ->enumNode('style')
+                            ->info('The style of the generated CSS: compressed or expanded (default: expanded)')
```

Comment:
no need to duplicate the default value in the `info` IMO (which risks becoming out of sync)